src/AppBundle/Entity/Comment.php

```diff
@@ -16,6 +16,7 @@
 
 /**
  * @ORM\Entity
+ * @ORM\Table(name="symfony_demo_comment")
```

Comment:
What about to use a table prefix for this? http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/cookbook/sql-table-prefixes.html
Or is it too advanced feature?